virttest/qemu_devices/qcontainer.py

```diff
@@ -1418,20 +1421,26 @@ def machine_other(cmd=False):
                                                (machine_type))
         else:
             devices = None
+            machine_opts = []
```

Comment:
The `else` blocker have existing issues, but the new added machine_accel logic is ok, so keep it is ok for me.